### PR TITLE
Fix GitHub Casing

### PR DESCRIPTION
**Pull request by varghesejose2020**

### CODE REVIEW

**Review by cwarnermm — APPROVED** [review]
Thank you, @varghesejose2020!

File: site/content/contribute/more-info/mobile/build-your-own/_index.md [hunk 1]
```diff
@@ -14,10 +14,13 @@ At Mattermost, we build and deploy the Apps using a CI pipeline. The pipeline ha
 
 As an alternative we've also created a set of **scripts** to help automate build tasks. Learn more about the scripts by reviewing the {{< newtabref href="https://github.com/mattermost/mattermost-mobile/blob/master/package.json" title="package.json" >}} file.
 
+---
 {{<note "Note:">}}
 **Note:** By using the **scripts**, {{< newtabref href="https://docs.fastlane.tools/#choose-your-installation-method" title="Fastlane" >}} and other dependencies will be installed in your system.
 {{<note "Note:">}}
```
Comment (neflyte) [inline site/content/contribute/more-info/mobile/build-your-own/_index.md:20]:
```suggestion
{{</note>}}
```
Please make sure to close the `note` tag at the end of the note. :slightly_smiling_face:

File: site/content/contribute/more-info/mobile/build-your-own/_index.md [hunk 2]
```diff
@@ -14,10 +14,13 @@ At Mattermost, we build and deploy the Apps using a CI pipeline. The pipeline ha
 
 As an alternative we've also created a set of **scripts** to help automate build tasks. Learn more about the scripts by reviewing the {{< newtabref href="https://github.com/mattermost/mattermost-mobile/blob/master/package.json" title="package.json" >}} file.
 
+---
```
Comment (neflyte) [inline site/content/contribute/more-info/mobile/build-your-own/_index.md:17]: Please remove this horizontal line.

File: site/content/contribute/more-info/mobile/build-your-own/_index.md [hunk 3]
```diff
@@ -14,10 +14,13 @@ At Mattermost, we build and deploy the Apps using a CI pipeline. The pipeline ha
...
 {{<note "Note:">}}
 **Note:** By using the **scripts**, {{< newtabref href="https://docs.fastlane.tools/#choose-your-installation-method" title="Fastlane" >}} and other dependencies will be installed in your system.
 {{<note "Note:">}}
 
+---
```
Comment (neflyte) [inline site/content/contribute/more-info/mobile/build-your-own/_index.md:22]: Please remove this horizontal line as well.

**Review by neflyte — CHANGES_REQUESTED** [review]

**Comment by neflyte** [comment]
Please make sure to test your changes locally or with the preview environment. Here's what I see in the preview:

image

**Comment by varghesejose2020** [comment]
Changes updated @cwarnermm

**Review by neflyte — APPROVED** [review]

**Comment by varghesejose2020** [comment]
@cwarnermm

File: site/content/contribute/more-info/mobile/build-your-own/_index.md [hunk 4]
```diff
@@ -16,7 +16,9 @@ As an alternative we've also created a set of **scripts** to help automate build
 
 {{<note "Note:">}}
 **Note:** By using the **scripts**, {{< newtabref href="https://docs.fastlane.tools/#choose-your-installation-method" title="Fastlane" >}} and other dependencies will be installed in your system.
-{{<note "Note:">}}
+{{</note>}}
```
Comment (cwarnermm) [inline site/content/contribute/more-info/mobile/build-your-own/_index.md:19]: @varghesejose2020 - Please delete the note syntax on line 19. The note title is set via line 17.

**Review by cwarnermm — CHANGES_REQUESTED** [review]
Thanks, @varghesejose2020! I've identified one line that should be removed from the commmitted changes. Once that change is applied, I can merge this PR.

Thank you for updating the copyright notice in the footer too! Very much appreciated!!

**Comment by varghesejose2020** [comment]
@cwarnermm  changes updated

File: site/content/contribute/more-info/mobile/build-your-own/_index.md [hunk 5]
```diff
@@ -16,7 +16,9 @@ As an alternative we've also created a set of **scripts** to help automate build
 
 {{<note "Note:">}}
 **Note:** By using the **scripts**, {{< newtabref href="https://docs.fastlane.tools/#choose-your-installation-method" title="Fastlane" >}} and other dependencies will be installed in your system.
```
Comment (cwarnermm) [inline site/content/contribute/more-info/mobile/build-your-own/_index.md:18]:
```suggestion
By using the **scripts**, {{< newtabref href="https://docs.fastlane.tools/#choose-your-installation-method" title="Fastlane" >}} and other dependencies will be installed in your system.
```

**Pull request merged**